automated install/basic-install.sh

```diff
@@ -167,6 +167,17 @@ for var in "$@"; do
     esac
 done
 
+if [[ "${runUnattended}" == true ]]; then
+    # In order to run an unattended setup, a pre-seeded /etc/pihole/pihole.toml must exist
+    if [[ ! -f "${PI_HOLE_CONFIG_DIR}/pihole.toml" ]]; then
+        printf "  %b Error: \"%s\" not found. Cannot run unattended setup\\n" "${CROSS}" "${PI_HOLE_CONFIG_DIR}/pihole.toml"
+        exit 1
```

Comment:
I know this PR was left in draft mode a few months ago, but the issue with `--unattended` on fresh installs is not solved.

What do you think about using `pihole-FTL create-default-config /etc/pihole/pihole.toml` to create a fresh config file, instead of `exit 1`? 

_https://github.com/pi-hole/FTL/blob/8d1add8d99d8a36ccc6fb97628a33a219cf8abbd/src/args.c#L674-L693_

```suggestion
        pihole-FTL create-default-config "${PI_HOLE_CONFIG_DIR}/pihole.toml"
```